### PR TITLE
ci: switch provisioning profile

### PR DESCRIPTION
**Pull request by carlh98**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Build or CI related changes

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

**Comment by GabrielChouinardLetourneau** [comment]
Are those changes necessary for other projects? Like Brighter for exemple?

**Comment by carlh98** [comment]
@GabrielChouinardLetourneau Brighter will need to switch to nventive.p12 this month

**Comment by GabrielChouinardLetourneau** [comment]
> 

cc @Guidemarcus

**Review by Soap-141 — APPROVED** [review]

**Review by Iguins — APPROVED** [review]

File: src/app/ApplicationTemplate.UITests/Constants.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@ namespace ApplicationTemplate.UITests;
 
 public static class Constants
 {
-	public const string IOSAppName = "com.nventive.internal.applicationtemplate";
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.UITests/Constants.cs:7]: Why remove `internal`???
Now iOS and Android don't match anymore.

File: build/variables.yml [hunk 2]
```diff
@@ -28,13 +28,13 @@ variables:
   # Make sure you have the following secured files in your Azure pipeline library.
   #
   # Android
-  InternalKeystore: com.nventive.internal.applicationtemplate.jks # This is the internal keystore used for internal builds.
-  GooglePlayKeystore: com.nventive.applicationtemplate.jks # This is the official keystore used for Google Play.
+  InternalKeystore: com.nventive.internal.applicationtemplate.jks # This should be the internal keystore used for internal builds.
+  GooglePlayKeystore: com.nventive.applicationtemplate.jks # This should be the official keystore used for Google Play.
   # iOS
-  InternalProvisioningProfile: com.nventive.internal.applicationtemplate.mobileprovision # This is the internal provisioning profile for internal builds.
```
Comment (jeanplevesque) [inline build/variables.yml:34]: Why change `is` to `should be`?

File: src/app/ApplicationTemplate.UITests/Constants.cs (hunk 1 above)
Comment (carlh98) [reply src/app/ApplicationTemplate.UITests/Constants.cs:7]: com.nventive.internal.applicationtemplate is in apple enterprise account, we can't use it in the individual account, for android and ios to match we'll have to change both of them.

**Pull request merged**